components/ui/ConditionalNuxtLink.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <nuxt-link v-if="isLink" :to="to">
+    <slot />
+  </nuxt-link>
+  <span v-else>
+    <slot />
+  </span>
+</template>
+
+<script setup>
+defineProps({
+  to: {
+    type: String,
+    required: true,
+  },
+  isLink: {
+    type: Boolean,
+    required: true,
+  },
+})
+</script>
```

Comment:
I don't think this needs to be a component. You can just use `<component :is="component" />

Reply:
think you forgot to finish writing this comment, not sure what you meant?

Reply:
component :is (lookup vue dynamic comments)